### ISSUE TITLE
[Question] Conformer pre-trained models

### ISSUE THREAD

**Issue by hoangtuanvu** [NVIDIA/NeMo#1408]
Thanks authors for publishing ASR sources. I know that QuartzNet & Jasper are comparable to some SOTA model for S2T tasks. Recently, Google published "Conformer" encoder network and achieved very good performance on LibriSpeech. I saw that, that network source is released on new Nemo version. Could you please release pre-trained model for this?

**Comment by LordOfLuck** [comment]
I would also be very interested!

**Comment by amirgholami** [comment]
Hi, first thanks for developing this repo. It is quite useful. I am also interested in the pretrained conformer model. It would be helpful if this could also be added to ngc

Thanks,

**Comment by VahidooX** [comment]
We have the plan to release publicly some checkpoints with Conformer-CTC, and maybe even Conformer-RNNT in our next release in March.

**Comment by amirgholami** [comment]
> We have the plan to release publicly some checkpoints with Conformer-CTC, and maybe even Conformer-RNNT in our next release in March.

Thanks Vahid, this would be very helpful

**Comment by hoangtuanvu** [comment]
Thanks @VahidooX

**Comment by rbracco** [comment]
I see there are now pretrained conformers for the CTCEncDecBPE models [here](https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/asr/results.html#automatic-speech-recognition-models). Are there any checkpoints anywhere for the char version? I found the yaml file for training conformer + char but can't seem to find any pretrained models or checkpoints. Thanks!

**Comment by VahidooX** [comment]
We do not have any plan to release any publicly available checkpoints trained with char-level encoding. In our experiments, accuracy of char-level models was worse than BPE-level models. Generally when you use downsampling, BPE-based models may work better. The current released checkpoints for Conformer-CTC are trained with just LS, and we have the plan to release some better checkpoints trained on larger datasets in the following month.

**Comment by sv711** [comment]
@VahidooX , any plans on releasing Conformer-RNNT pre-trained weights? It would be helpful if you could share any sort of rough timeline as to when can we expect it.

**Comment by VahidooX** [comment]
Conformer-CTC checkpoints published:
https://ngc.nvidia.com/catalog/collections/nvidia:nemo_asr
